app/datasources/db/models.py

```diff
@@ -283,10 +284,12 @@ async def get_abi_by_contract_address(
     @classmethod
     async def get_contracts_without_abi(
         cls, session: AsyncSession, max_retries: int = 0
-    ):
+    ) -> AsyncIterator[Row[tuple[Self]]]:
```

Comment:
I'm changing the return value here https://github.com/safe-global/safe-decoder-service/pull/64/files#diff-77a1b67cebbe84329bc4e9a69e248a5e7a50b5f8ba8911b68b6b26ab488aa695R286 
Basically to don't return a tuple. And just an AsyncGenerator with Contract.
Feel free to update your changes accordingly if you want.

Reply:
I cannot update your changes until your PR is merged, so I will merge this and then you can rebase it